#################################################################
#								#
# Copyright (c) 2021-2022 YottaDB LLC and/or its subsidiaries.	#
# All rights reserved.						#
#								#
# Original code by Zachary Minneker from Security Innovation.	#
#								#
#	This source code contains the intellectual property	#
#	of its copyright holder(s), and is made available	#
#	under a license.  If you do not know the terms of	#
#	the license, please stop and do not read further.	#
#								#
#################################################################

diff --git a/sr_unix/sig_init.c b/sr_unix/sig_init.c
index 9b986ed7..42a57471 100755
--- a/sr_unix/sig_init.c
+++ b/sr_unix/sig_init.c
@@ -147,7 +147,7 @@ void sig_init(void (*signal_handler)(), void (*ctrlc_handler)(), void (*suspsig_
 			case SIGTERM:
 			case SIGTRAP:
 				/* These are all being handled by the generic_signal_handler. */
-				sigaction(sig, &gen_action, NULL);
+				//sigaction(sig, &gen_action, NULL);
 				break;
 			case SIGKILL:
 			case SIGSTOP:
